auto_round/autoround.py

```diff
@@ -150,6 +151,8 @@ def __init__(
         self.quantized = False
         self.model_orig_dtype = model.dtype
         self.low_cpu_mem_usage = low_cpu_mem_usage
+        assert not is_mixed_device(model), ("autoround not support for mixed device mapping," 
+                "please put the model to cpu or set low_cpu_mem_usage to True if want to save memory.")
```

Comment:
1 remove "if want to save memory"
2present the exact cmd "model to cpu", could low_cpu_mem solve this issue, please have a check. if not, remove it.